etl/steps/data/garden/irena/2023-12-12/renewable_power_generation_costs.countries.json

```diff
@@ -14,10 +15,8 @@
     "Republic of Korea": "South Korea",
     "Spain": "Spain",
     "Sweden": "Sweden",
-    "Ukraine": "Ukraine",
     "United Kingdom": "United Kingdom",
     "United States": "United States",
-    "Viet Nam": "Vietnam",
```

Comment:
do you know why the new version doesn't include the data for Vietnam and Ukraine?

Reply:
Good call, I think I've just figured it out: They added those (and a few others) as a separate sheet, for smaller markets... I'll try to combine them.